### ISSUE TITLE
Incomplete data are not displayed with `ggmice()` if log transformation is applied within the mapping

### ISSUE THREAD

**Issue by hanneoberman** [amices/ggmice#34]
One solution is to apply the transformation beforehand. Maybe just add a message whenever a log transformation is applied on an incomplete variable within the aes() function? Or actually solve the issue?

**Comment by hanneoberman** [comment]
There 2 options:
- data -> transform with mutate -> ggmice
- data -> ggmice -> scale_*_log

**Comment by hanneoberman** [comment]
``` r
ggmice(dat, ggplot2::aes(x = age, y = log(bmi))) + ggplot2::geom_point()
#> Error in ggmice(dat, ggplot2::aes(x = age, y = log(bmi))): could not find function "ggmice"
```

<sup>Created on 2023-01-19 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by hanneoberman** [comment]
@pepijnvink please write a message if `log(` is included in one of the mapping variables

**Comment by hanneoberman** [comment]
Fixed in #80
